src/main/app/src/app/zoeken/model/zoek-parameters.ts

```diff
@@ -21,8 +21,8 @@ export class ZoekParameters {
             tot: null
         }
     };
-    sorteerVeld: string;
-    sorteerRichting: string;
-    rows: number;
-    start: number;
+    sorteerVeld: string = 'IDENTIFICATIE';
```

Comment:
Ik weet niet of het een goed idee is maar je zou hier, net als in het veld hieronder, alle mogelijke waarden kunnen opgeven. Dan worft het meer type safe. Of van IDENTIFICATIE ergens een constante maken of een enum?

Reply:
Ik laat het eerst even zo, zoek en filtervelden hebben ook geen types (is niet mogelijk met indexed arrays)